packages/vuetify/src/services/theme/index.ts

```diff
@@ -0,0 +1,223 @@
+// Extensions
+import { Service } from '../service'
+
+// Utilities
+import * as ThemeUtils from './utils/theme'
+
+// Types
+import { VuetifyParsedTheme, VuetifyThemeOptions } from 'vuetify/types/services/theme'
+
+export class Theme extends Service {
+  static property = 'theme'
+
+  public dark = false
+  public disabled = false
+  public options: VuetifyThemeOptions['options']
+  public styleEl?: HTMLStyleElement
+  public themes: VuetifyThemeOptions['themes'] = {
+    dark: { // Maybe use variables here?
```

Comment:
I like the idea to using variables. By doing so, colors can be changed in one place.